app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+end
```

Comment:
- [ ] As requested by the project requirements, the Comment model should have a method that updates the comments counter for a post. Kindly add this.